core/src/main/java/io/questdb/griffin/engine/functions/bind/BooleanBindVariable.java

```diff
@@ -62,4 +65,9 @@ public boolean isThreadSafe() {
     public void toPlan(PlanSink sink) {
         sink.val("?::boolean");
     }
+
+    @Override
+    public void toSink(@NotNull CharSink<?> sink) {
+        sink.put(value ? "true" : "false").put("::boolean");
```

Comment:
sink already implements `put(boolean)`, so it could be just `sink.put(value).put("::boolean")`